scripts/oauth/token.go

```diff
@@ -272,7 +272,7 @@ func setup() *OAuthApp {
 
 	substitutions, err := registry.GetMap(SubstitutionsFieldName)
 	if err != nil {
-		slog.Warn("no substitutions, ensure that the provider info doesn't have any {{variables}}", err)
```

Comment:
This suddenly caused test failings(building was failing), Updated this to resolve the  it.